rust-escrow/conditional-escrow/src/lib.rs

```diff
@@ -127,7 +142,15 @@ impl ConditionalEscrow {
         let payee = self.get_recipient_account_id();
```

Comment:
Let's change this to: 

```suggestion
        let recipient_contract_id = self.get_recipient_account_id();
```